src/components/ArtistList.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import List from 'react-toolbox/lib/list/List';
+import ListSubHeader from 'react-toolbox/lib/list/ListSubHeader';
+import ArtistListItem from '../ArtistListItem';
+
+class ArtistList extends Component {
+  render() {
```

Comment:
Future: Always use a pure functional component if there is no state. Especially under Redux, being purely functional and prop-driven becomes increasingly important.